### PR TITLE
Change title color

### PR DESCRIPTION
**Pull request by bernardini687**
hello teammates I've picked a nice red for the heading in the root page

### CODE REVIEW

**Comment by gregoriofracassi** [comment]
that red is unreal man

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -1 +1,4 @@
 // Specific CSS for your home-page
+.red-title {
+  color: #ff4040;
```
Comment (kmontes417) [inline app/assets/stylesheets/pages/_home.scss]: I like red

Comment (kmontes417) [reply app/assets/stylesheets/pages/_home.scss]: can you change it to green?

Comment (Adamsonson) [reply app/assets/stylesheets/pages/_home.scss]: green like pickle! yes please

File: app/assets/stylesheets/pages/_home.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Specific CSS for your home-page
-.red-title {
-  color: #ff4040;
+.green-title {
```
Comment (Adamsonson) [inline app/assets/stylesheets/pages/_home.scss]: green like pickle, I like it!

**Pull request merged**